packages/ember-debug/lib/warn.js

```diff
@@ -18,16 +18,6 @@ export let missingOptionsDeprecation = 'When calling `Ember.warn` you ' +
   '`options` should include an `id` property.';
 export let missingOptionsIdDeprecation = 'When calling `Ember.warn` you must provide `id` in options.';
 
-/**
-  Display a warning with the provided message. Ember build tools will
-  remove any calls to `Ember.warn()` when doing a production build.
-
-  @method warn
-  @param {String} message A warning to display.
-  @param {Boolean} test An optional boolean. If falsy, the warning
-    will be displayed.
-  @public
```

Comment:
I believe that we need to add the following to the beginning of this file:

```js
/**
@module ember
@submodule ember-debug
*/
```

along with `@for Ember` right above `@public`.